Replace eos token: <|im_end|>
Replace eos token: <|im_end|>
Replace eos token: <|im_end|>
Replace eos token: <|im_end|>

  3%|██▊                                                                                       | 1/32 [00:21<11:20, 21.96s/it]
Replace eos token: <|im_end|>
  3%|██▊                                                                                       | 1/32 [00:21<11:20, 21.96s/it]Traceback (most recent call last):
  File "/Users/a58/Downloads/MoE_Train/finetune/run.py", line 163, in <module>
    main()
  File "/Users/a58/Downloads/MoE_Train/finetune/run.py", line 153, in main
    trainer.train()
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/transformers/trainer.py", line 1624, in train
    return inner_training_loop(
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/transformers/trainer.py", line 1961, in _inner_training_loop
    tr_loss_step = self.training_step(model, inputs)
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/transformers/trainer.py", line 2902, in training_step
    loss = self.compute_loss(model, inputs)
  File "/Users/a58/Downloads/MoE_Train/finetune/trainer.py", line 47, in compute_loss
    outputs = model(**inputs)
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/peft/peft_model.py", line 1129, in forward
    return self.base_model(
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/peft/tuners/tuners_utils.py", line 161, in forward
    return self.model.forward(*args, **kwargs)
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/transformers/models/qwen2/modeling_qwen2.py", line 1186, in forward
    logits = self.lm_head(hidden_states)
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/Users/a58/anaconda3/envs/llama_factory/lib/python3.10/site-packages/torch/nn/modules/linear.py", line 114, in forward
    return F.linear(input, self.weight, self.bias)
RuntimeError: MPS backend out of memory (MPS allocated: 14.05 GB, other allocations: 3.33 GB, max allowed: 18.13 GB). Tried to allocate 1.11 GB on private pool. Use PYTORCH_MPS_HIGH_WATERMARK_RATIO=0.0 to disable upper limit for memory allocations (may cause system failure).